src/Executables/ExportCoordinates/ExportCoordinates.hpp

```diff
@@ -110,8 +114,66 @@ struct ExportCoordinates {
     return std::forward_as_tuple(std::move(box));
   }
 };
+
+struct PrintGlobalMinimumGridSpacing {
+  template <typename ParallelComponent, typename DbTags, typename Metavariables,
+            typename ArrayIndex>
+  static void apply(const db::DataBox<DbTags>& /*box*/,
+                    const Parallel::GlobalCache<Metavariables>& /*cache*/,
+                    const ArrayIndex& /*array_index*/, const double time,
+                    const double global_min_grid_spacing) noexcept {
+    Parallel::printf("Time: %f, Global inertial minimum grid spacing: %f\n",
```

Comment:
I'd suggest using `%1.3e` instead of `%f` so that we always get 4 digits regardless of the scale (printing in C++ is silly...)

Reply:
Done!